packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -265,7 +267,8 @@ abstract class OAuth2Client {
     init = init || ({} as RequestInit);
 
     if (includeToken) {
-      const { accessToken } = await TokenStorage.get();
+      const tokens = await TokenStorage.get();
+      const accessToken = tokens && tokens.accessToken;
```

Comment:
This helps the code with resiliency. Now, tokens returning `null` doesn't crash.